deployer/src/persistence/deployment.rs

```diff
@@ -86,7 +86,6 @@ pub trait DeploymentUpdater: Clone + Send + Sync + 'static {
 pub struct DeploymentState {
     pub id: Uuid,
     pub state: State,
-    pub last_update: DateTime<Utc>,
```

Comment:
Why are we dropping this?

Reply:
It is added when the table updates instead. (other comment)